packages/react-components/react-avatar/src/components/AvatarGroupItem/AvatarGroupItem.tsx

```diff
@@ -6,7 +6,8 @@ import type { AvatarGroupItemProps } from './AvatarGroupItem.types';
 import type { ForwardRefComponent } from '@fluentui/react-utilities';
 
 /**
- * AvatarGroupItem component - TODO: add more docs
+ * The AvatarGroupItem component represents a single person or entity.
+ * AvatarGroupItem should only be used in an AvatarGroup component.
```

Comment:
[This can be done in a future PR and doesn't need to be resolved in this one]

This comment made me think we might want to add some development-only console warnings (under a `process.env.NODE_ENV !== 'production'` check) if:
* Avatar is used as a child of AvatarGroup (check the children array for items of type Avatar)
* AvatarGroupItem is used outside an AvatarGroup (check if there's no AvatarGroupContext)